test/inductor/test_torchinductor.py

```diff
@@ -1727,6 +1726,8 @@ def test(
             fn_opt = torch.compile(fn)
             if is_halide_backend(self.device):
                 pass  # no device asserts in halide
+            if is_mps_backend(self.device):
+                pass  # no device asserts in MPS
```

Comment:
Do you mind adding TODO there, we should enable them, but this is gated on https://github.com/pytorch/pytorch/issues/144634

Reply:
Added, thanks.